### ISSUE TITLE
Captions won't save, kind of

### ISSUE THREAD

**Issue by shyatt** [BYU-ARCLITE/Ayamel-Examples#471]
The last bug I submitted was not 100% correct. The captions saved, but only after I logged out of Ayamel and then logged back on. It was the same for Firefox and Chrome.

**Comment by gliese1337** [comment]
Duplicate of #470
